src/main/java/CommandLineUI.java

```diff
@@ -98,6 +98,11 @@ public void displayBoard(List<Mark> grid, int size) {
         output.println(formattedRows);
     }
 
+    @Override
+    public void announceGameStatus(Result winner) {
+
```

Comment:
Do you need to Override despite you don't implement anything?

Reply:
Good point, thank you Fabien :)